### ISSUE TITLE
Unable to use array as a path parameter

### ISSUE THREAD

**Issue by lucasprograms** [cdimascio/express-openapi-validator#305]
**Describe the bug**
Unable to use array as a path parameter.

**To Reproduce**
Schema:

```
openapi: 3.0.1
info:
  version: 2.0.0a
  title: Test API
  description: |
    # Welcome!
servers:
  - url: /v2
    description: All URLs in this specification use the '/v2' endpoint subdirectory.
tags:
  - name: Feature APIs
paths:
  /features/{uuids}:
    delete:
      summary: Deletes Features given a number of uuids.
      parameters:
        - in: path
          name: uuids
          required: true
          schema:
            type: array
            items:
              type: string
          style: simple
      responses:
        '200':
          description: Features were successfully deleted
          content:
            application/json:
              schema:
                type: object
```

Request:
```
DELETE my-server.biz/v2/features/40183e0f-295b-4d35-93e4-9a70e4dd8b1a,6643b200-6642-4ff5-808b-72fa54dab7f4
```

**Actual behavior** 

400 Error:
```
{
    "message": "Parameter 'style' has incorrect value 'simple' for [uuids]",
    "errors": [
        {
            "path": ".query.uuids",
            "message": "Parameter 'style' has incorrect value 'simple' for [uuids]"
        }
    ]
}
```

**Expected behavior**
Request is not considered to be "bad"

Is there something obviously wrong with my openapi spec? I cannot get any array path parameters to work.

**Comment by cdimascio** [comment]
@lucasprograms thanks for the issue. this needs to be fixed

**Comment by cdimascio** [comment]
@lucasprograms this should be fixed in `v3.14.1`

**Comment by lucasprograms** [comment]
@cdimascio Thank you very, very much!
